test/e2e/framework/statefulset/wait.go

```diff
@@ -121,6 +121,31 @@ func WaitForStatusReadyReplicas(c clientset.Interface, ss *appsv1.StatefulSet, e
 	}
 }
 
+// WaitForStatusAvailableReplicas waits for the ss.Status.Available to be equal to expectedReplicas
+func WaitForStatusAvailableReplicas(c clientset.Interface, ss *appsv1.StatefulSet, expectedReplicas int32) {
+	framework.Logf("Waiting for statefulset status.AvaialbleReplicas updated to %d", expectedReplicas)
```

Comment:
nit: naming format of `statefulset status.AvaialbleReplicas` could be logged consistently in this function